client/overview/style.scss

```diff
@@ -33,9 +33,24 @@
 	}
 
 	.wcpay-connection-success {
+		position: relative;
 		text-align: center;
 		padding: $gap-large;
-		padding-bottom: $gap;
+
+		&__dropdown {
+			position: absolute;
+			top: 0 + $gap-large;
+			right: 0 + $gap-large;
+
+			.dashicons-ellipsis {
+				transform: rotate( 90deg );
+			}
+		}
+
+		.dashicons-button {
+			margin-right: 0;
+			visibility: hidden;
+		}
```

Comment:
Maybe using just `display: none` will be better to avoid that extra margin on the left. It looks a bit odd right now. What do you think?

| Before | After |
| - | - |
|<img width="160" alt="Screenshot 2023-04-11 at 10 18 57" src="https://user-images.githubusercontent.com/7670276/231100554-55b109a2-8d37-4208-bdd6-52271fff4939.png">|<img width="110" alt="Screenshot 2023-04-11 at 10 22 38" src="https://user-images.githubusercontent.com/7670276/231100694-678627b9-b53b-498d-98e0-86bc655a9004.png">|

Reply:
I tried before and it was not being applied. I've had to use `!important` here, but I think it makes it look better  https://github.com/Automattic/woocommerce-payments/pull/5992/commits/237bc0adb39a072bfbdeee4d5d9670b93471a342